src/data/palettes1.asm

```diff
@@ -0,0 +1,667 @@
+Palette0:: ; b738a (2d:738a)
+	db 1, $e4, 8
+	db $9c, $63, $94, $42, $08, $21, $00, $00

```

Comment:
I'm assuming this format is compatible with the existing `rgb` macro?

Reply:
Indeed!